packages/cadl-ranch-specs/http/models/property-types/main.tsp

```diff
@@ -183,3 +183,50 @@ model NeverProperty is ModelTemplate<never>;
 @route("/never")
 @operationGroup
 interface Never extends ModelOperations<NeverProperty, "<don't include this property>"> {}
+
+// Test a model with a model property which could be nullable
+@doc("Model with nullable properties")
+model ModelWithNullableProperty {
+  @doc("id property")
+  id: string;
+
+  @doc("nullable property")
+  property: string | null;
```

Comment:
Sorry to bring new things but do we only care to test string or should we have a larger coverage of types that can be `null` as property.
i.e. Should we have a dedicated nullable model property spec that is pretty much like the optional model property spec but with `| null` instead of `?:`